### ISSUE TITLE
Build fail.

### ISSUE THREAD

**Issue by brandimarte** [tfrederiksen/inelastica#25]
Inelastica will not build if only the source code is downloaded (without cloning the repo).

```bash
Traceback (most recent call last):
  File "setup.py", line 171, in <module>
    write_version()
  File "setup.py", line 169, in write_version
    description=GIT_DESCR))
IndexError: list index out of range
```

**Comment by brandimarte** [comment]
Fixed submited bc6d30f

**Comment by brandimarte** [comment]
49b4ed2 fix execution without version information.
